api/src/routes/guilds/templates/index.ts

```diff
@@ -1,11 +1,10 @@
 import { Request, Response, Router } from "express";
-const router: Router = Router();
 import { Template, Guild, Role, Snowflake, Config, User, Member } from "@fosscord/util";
-const { enabled, allowTemplateCreation, allowDiscordTemplates, allowRaws } = Config.get().templates;
 import { route } from "@fosscord/api";
 import { DiscordApiErrors } from "@fosscord/util";
 import fetch from "node-fetch";
-
+const router: Router = Router();
+const { enabled, allowTemplateCreation, allowDiscordTemplates, allowRaws } = Config.get().templates;
```

Comment:
Move the config checks inside the routes: https://docs.fosscord.com/contributing/server/#example

Reply:
You need to put this inside of the router.post(...) handler